pep-0649.rst

```diff
@@ -713,15 +728,18 @@ makes some modifications to the annotations before it returns them.
 This PEP adds a new keyword-only parameter to these two functions,
 ``format``.  ``format`` specifies what format the values in the
 annotations dict should be returned in.
-``format`` accepts the following values, defined as attributes on the
+``format`` accepts the following values, as defined the
 ``inspect`` module::
 
-    VALUE = 1
-    FORWARDREF = 2
-    SOURCE = 3
+    class AnnotationFormat(enum.IntEnum):
+        VALUE = 1
+        FORWARDREF = 2
+        SOURCE = 3
+    VALUE = AnnotationFormat.VALUE
+    FORWARDREF = AnnotationFormat.FORWARDREF
+    SOURCE = AnnotationFormat.SOURCE
```

Comment:
You could consider using ``enum.global_enum`` here perhaps ([reference](https://docs.python.org/3/library/enum.html#enum.global_enum)), though this would change the repr from ``<AnnotationFormat.VALUE: 1>`` to ``inspect.VALUE``.

Reply:
Thanks!  I hadn't heard of that.  I will probably do it.  But I don't think it has to go in the PEP.  I think I'd prefer to omit it from the PEP, actually, just to keep it light, breezy reading.

Reply:
I can't get it to work.  If I add a line above "class AnnotationFormat", indented at the same level as "class", that reads "@enum.global_enum", Sphinx:

`sphinx.errors.SphinxParallelError: docutils.utils.SystemMessage: /home/larry/src/python/peps/larrytrunk/pep-0649.rst:751: (ERROR/3) Unexpected indentation.`

If I screwed up the formatting, fair enough, but I don't think I did.  I think this is a Sphinx bug.

Can you get it to work?  I'm leaving it out for now.

Reply:
> Can you get it to work?

I've gotten it to work, but can't do a suggestion via GH's interface -- do you mind if I push a commit to fix?

Reply:
Please do!

Reply:
Done!